exodus/analysis_query/templates/query_submit.html

```diff
@@ -50,6 +50,12 @@ <h1 class="main-title">
           </div>
         </div>
 
+        {% if upload_allowed %}
+        <p class="text-right">
+          <a class="link main-link" href="{% url 'analysis:upload' %}">{% trans "I prefer to upload an application." %}</a>
+        </p>
```

Comment:
1) Right indent the block
2) Remove the dot at the end of the link?

Reply:
Okay